### PR TITLE
add some troubleshooting commands

### PR DESCRIPTION
**Pull request by sean-harness**

### CODE REVIEW

File: docker-compose/harness/README.md [hunk 1]
```diff
@@ -32,6 +32,25 @@ This page contains the instructions to run Harness CD Community Edition using Do
...
+```shell
+docker-compose logs -f <NAME>
+```
+For example,
+```shell
+docker-compose logs -f manager
+```
+
+## Support
+[Community forums](https://community.harness.io/)
```
Comment (schoudhury) [inline docker-compose/harness/README.md:51]: [Join the Harness Community Forum](https://community.harness.io/)

File: docker-compose/harness/README.md [hunk 2]
```diff
@@ -32,6 +32,25 @@ This page contains the instructions to run Harness CD Community Edition using Do
...
+docker-compose logs -f <NAME>
+```
+For example,
+```shell
+docker-compose logs -f manager
+```
+
+## Support
+[Community forums](https://community.harness.io/)  
+[Community Slack channel](https://harnesscommunity.slack.com/archives/C02K03Q5L0J)
```
Comment (schoudhury) [inline docker-compose/harness/README.md:52]: [Join the Harness Community Slack](https://join.slack.com/t/harnesscommunity/shared_invite/zt-y4hdqh7p-RVuEQyIl5Hcx4Ck8VCvzBw)

File: helm/README.md [hunk 3]
```diff
@@ -44,6 +44,20 @@ kubectl get services -n harness
...
+```shell
+kubectl get pods -n harness
+```
+##### View logs of processes
+```shell
+kubectl logs -n harness -f <POD-NAME>
+```
+
+## Support
+[Community forums](https://community.harness.io/)
```
Comment (schoudhury) [inline helm/README.md:59]: same as above

**Review by schoudhury — CHANGES_REQUESTED** [review]

File: docker-compose/harness/README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ This page contains the instructions to run Harness CD Community Edition using Do
 * Internet connection
 
 ## Start Harness CD
-1) Increase Docker Desktop memory and CPU by the numbers listed above. For Docker Desktop, see [Docker for Mac](https://docs.docker.com/docker-for-mac/#resources) or [Docker for Windows](https://docs.docker.com/docker-for-windows/#resources) for details on increasing resources.
+1) For Windows and Mac, increase Docker Desktop memory and CPU by the numbers listed above. See [Docker for Mac](https://docs.docker.com/docker-for-mac/#resources) or [Docker for Windows](https://docs.docker.com/docker-for-windows/#resources) for details on increasing resources.
```
[the remark this answers is not in the document]
Comment (sean-harness) [reply docker-compose/harness/README.md:14]: This is made on top of your change, it was due to some feedback where a user got the impression that we do not support Linux, so I wanted to clarify this step is only applicable for Windows and Mac users.

**Review by gunascode — APPROVED** [review]

**Pull request merged**